docs/upgrade/1-11.0.md

```diff
@@ -10,3 +10,35 @@ nav_order: 2
 Open Distro for Elasticsearch 1.11.0 renames the SQL Workbench plugin for Kibana to the Query Workbench.
 
 Instead of incrementing the `opendistro_sql_workbench` version number, remove the plugin and install `opendistro-query-workbench` instead, as covered in [Standalone Kibana plugin install](../../kibana/plugins/).
+
+## Alerting and anomaly detection
+
+If you’re upgrading from an older ODFE version and you've enabled filter by backend roles with the following setting:
+
+```json
+PUT _cluster/settings
+{
+  "transient": {
+    "opendistro.alerting.filter_by_backend_roles": "true"
+  }
+}
+```
+
+Make sure you’re aware of the following:
+
+### Previously created detectors / monitors
+
+For detectors or monitors created prior to the upgrade, only admin users can assign permissions.
+
+After upgrading to the new ODFE version, we recommend that all admin users must update their existing detectors or monitors using the [Update API](../api/#update-detector). If not, the detectors or monitors will not be visible to any user, including that admin user.
+
+You only need to do this once and don’t have to repeat it for future upgrades.
+
+### Users with no backend roles
+
+All users must have an assigned backend role. If a user doesn’t have any backend role, the user cannot create detectors or monitors even with the correct permissions. This is because users are first filtered by the backend role.
+
+### Backward compatibility
```

Comment:
Surely there's a workaround. Recreate the AD indices? Update your detectors...?

Reply:
Definitely, recreating AD indices will solve the problem but customers will loose existing detectors. 
Updating detectors will not fix it due to an edge case. We fixed this for the next ODFE release.

Reply:
Had an offline chat with Ashwin, we are good to go.